system/Pager/PagerRenderer.php

```diff
@@ -83,17 +83,17 @@ class PagerRenderer
     protected $pageSelector;
 
     /**
-     * The number of items a page.
+     * The maximum number of items displayed per page.
```

Comment:
See `Pager->getPerPage()`:
```suggestion
     * Returns the number of results per page that should be shown.
```